plugins/woocommerce-admin/client/marketplace/stylesheets/_variables.scss

```diff
@@ -11,12 +11,14 @@ $content-spacing-small: $grid-unit-20;
 $content-spacing-large: $grid-unit-40;
 $content-spacing-small: $medium-gap;
 $content-spacing-large: $xlarge-gap;
+$content-spacing-xlarge: $grid-unit-60;
 $content-max-width: 1600px;
 
 // Breakpoints
 $breakpoint-medium: 769px;
 $breakpoint-large: 1024px;
 $breakpoint-xlarge: 1500px;
+$breakpoint-huge: 1920px;
```

Comment:
Using the [Gutenberg xhuge breakpoint](https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_breakpoints.scss#L6) value to display 4 product cards in a row on wider screens.